<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="submit">
   <property name="geometry">
    <rect>
     <x>633</x>
     <y>580</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:pink;</string>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QToolButton" name="select_path">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>100</y>
     <width>26</width>
     <height>24</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>Select an Image</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="mildew_select">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>480</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Powdery Mildew</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rust_select">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>480</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Rust</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>50</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose an image</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="path">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>100</y>
     <width>531</width>
     <height>25</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>UpArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="image_original">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>150</y>
     <width>281</width>
     <height>251</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../Major Project/mildew/squash_powdery_mildew/1.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
